spec/features/merchant/items/index_spec.rb

```diff
@@ -21,7 +21,7 @@
         expect(page).to have_content("Price: $#{@tire.price}")
         expect(page).to have_css("img[src*='#{@tire.image}']")
         expect(page).to have_content("Active")
-        expect(page).to_not have_content(@tire.description)
```

Comment:
I went back and looked at this earlier after we had talked briefly and I did the same thing. Removed the condition from the view as well